### PR TITLE
move examples around multiagent, multi receiver, python, and scripts

### PR DESCRIPTION
**Pull request by davidiw**
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4703)

### CODE REVIEW

File: ecosystem/python/sdk/aptos_sdk/transactions.py [hunk 1]
```diff
@@ -207,14 +207,95 @@ def serialize(self, serializer: Serializer):
 
 
 class Script:
-    def __init__(self):
-        raise NotImplementedError
+    code: bytes
+    ty_args: List[TypeTag]
+    args: List[ScriptArgument]
+
+    def __init__(self, code: bytes, ty_args: List[TypeTag], args: List[bytes]):
```
Comment (jjleng) [inline ecosystem/python/sdk/aptos_sdk/transactions.py:214]: why not `List[ScriptArgument]` for `args` here?

File: aptos-move/move-examples/scripts/two_by_two_transfer/sources/script.move [hunk 2]
```diff
@@ -0,0 +1,28 @@
+script {
+    use aptos_framework::aptos_coin;
+    use aptos_framework::coin;
+
+    // Tune this parameter based upon the actual gas costs
+    const GAS_BUFFER: u64 = 100000;
+    const U64_MAX: u64 = 18446744073709551615;
```
[the remark this answers is not in the document]
Comment (davidiw) [reply aptos-move/move-examples/scripts/two_by_two_transfer/sources/script.move:7]: muahahaha, yes, good call!

File: ecosystem/python/sdk/aptos_sdk/transactions.py (hunk 1 above)
Comment (davidiw) [reply ecosystem/python/sdk/aptos_sdk/transactions.py:214]: Sigh... I really wish Python typing did something automatically

**Review by davidiw — COMMENTED** [review]
thanks for the review!

**Review by gregnazario — APPROVED** [review]

**Pull request merged**